packages/trpc/server/auth-router/router.ts

```diff
@@ -12,13 +12,18 @@ export const authRouter = router({
 
       return await createUser({ name, email, password, signature });
     } catch (err) {
-      console.error(err);
-
-      throw new TRPCError({
-        code: 'BAD_REQUEST',
-        message:
-          'We were unable to create your account. Please review the information you provided and try again.',
-      });
+      if (err instanceof Error && err.message === 'User already exists') {
+        throw new TRPCError({
+          code: 'BAD_REQUEST',
+          message: 'User with this email already exists. Please use a different email address.',
+        });
+      } else {
+        throw new TRPCError({
+          code: 'BAD_REQUEST',
+          message:
+            'We were unable to create your account. Please review the information you provided and try again.',
+        });
+      }
```

Comment:
I think doing something like this would be more optimal as it would allow us to add more errors in the future if need be:

```
let message = 'We were unable to create your account. Please review the information you provided and try again.';

if (err instanceof Error && err.message === 'User already exists') {
  message = 'User with this email already exists. Please use a different email address.';
}

throw new TRPCError({
  code: 'BAD_REQUEST',
  message,
});
```

Reply:
Sure, I’ll make the changes

Reply:
Made the changes, please have a look @dguyen